app/controllers/marcas_controller.rb

```diff
@@ -1,21 +1,43 @@
-class MarcasController < ApplicationController
-    def index
-        @random_phone = random
-    end
+require "rest-client"
+require "json"
+require_relative "./api_key"
+class MarcasController < OnlyphonesController
 
-    def random
-        rd = rand 1..2597
-        random_phone = request_api(rd)
-        name = random_phone['data']['name']
-        price = random_phone['data']['prices'][0]['price']
-        img = random_phone['data']['images'][0]['url']
-        [name, price, img]
-    end
+  def index
+    @phone_brand1 = brands("Apple") 
+    @phone_brand2 = brands("Samsung")
+    @phone_brand3 = brands("Xiaomi")
+    @phone_brand4 = brands("Oppo")
+    @phone_brand5 = brands("Motorola")
+    @phone_brand6 = brands("Google")
+    @phone_brand7 = brands("Huawei") 
+  end

```

Comment:
I think it would be a better idea to use enums in this part.
```suggestion
  def index
    enum :phone_brands, { apple: 0, samsung: 1, xiaomi: 2, motorola: 3, oppo: 4, google: 5, huawei: 6 }
  end

```

Reply:
Changes applied to the code based on the feedback.